product_demand_variability/models/product.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 ForgeFlow S.L.(http://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+import numpy as np
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    profile_categ_id = fields.Many2one("variability.profile")
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    profile_id = fields.Many2one(comodel_name="variability.profile")
```

Comment:
Hi, thanks for this nice module/features
Probably profile is a so generic name than there is colision with fields on other module like product_profile in v10/12
Variability seems more precise